shared/wallets/send-form/index.stories.tsx

```diff
@@ -23,12 +23,12 @@ const provider = banner =>
       amountErrMsg: '',
     }),
     Banner: props => ({}),
-    ConnectedPublicMemo: props => ({onChangePublicMemo: Sb.action('onChangePublicMemo')}),
+    ConnectedPublicMemo: props => ({onChangePublicMemo: Sb.action('onChangePublicMemo'), maxLength: 28}),
```

Comment:
I noticed that the memo and secret notes for the send form story were no longer getting character limits, since that data was changed to be provided by redux state in #18315.